code/modules/research/server.dm

```diff
@@ -1,5 +1,6 @@
 /obj/machinery/rnd/server
 	name = "R&D Server"
```

Comment:
This is also not properly capitalized, so it might be a good idea to make it like "research & development server" or something, since otherwise it'll display as "That's R&D Server."

Reply:
`name = "\improper R&D server"`